src/core/server/types.ts

```diff
@@ -44,4 +44,4 @@ export type {
 } from './saved_objects/service';
 export type { DomainDeprecationDetails, DeprecationsGetResponse } from './deprecations/types';
 export * from './ui_settings/types';
-export type { ExternalUrlConfig } from './http';
+export type { ExternalUrlConfig } from '@kbn/core-http-server-internal';
```

Comment:
I didn't want to remove this export in the current PR to avoid additional files in the diff. `src/core/server/types.ts` should be removed as a whole as the cleanup task.